packages/react/src/components/SFS/SFS.stories.tsx

```diff
@@ -48,34 +48,12 @@ const meta: Meta<Args> = {
 };
 
 const SFSChipsTooltipProps = {
-  slotProps: {
-    popper: {
-      modifiers: [
-        {
-          name: 'offset',
-          options: {
-            offset: [0, -4],
-          },
-        },
-      ],
-    },
-  },
+  distance: 4,
```

Comment:
```suggestion
  distance: 2,
```
https://www.notion.so/elonsoft/SFS-67268f1dc88946318382b62d6f7eb4b7?pvs=4#1a7814cebae08088afb7fe0f957d7bfc